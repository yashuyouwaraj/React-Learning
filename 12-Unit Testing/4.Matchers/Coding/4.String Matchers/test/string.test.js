import {it,expect} from "vitest"

it("toBe Matcher for strings",()=>{
    const result="Hello, World!"
    expect(result).toBe("Hello, World!")
})

it("toContain matcher for strings",()=>{
    const result="Hello, World!"
    expect(result).toContain("Hello")
})

it("toMatch Matcher for regex",()=>{
    const result="Hello, world!"
    expect(result).toMatch(/world/)
})

it("toMatchSnapshot matcher",()=>{
    const result="Hello, world!"
    expect(result).toMatchSnapshot()
})