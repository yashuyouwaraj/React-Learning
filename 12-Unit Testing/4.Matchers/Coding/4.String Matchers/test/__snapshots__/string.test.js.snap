// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`toMatchSnapshot matcher 1`] = `"Hello, world!"`;
